funnel/models/project.py

```diff
@@ -387,17 +393,29 @@ def calendar_weeks(self):
         #         Session.scheduled
         #         ).group_by(db.text('date')).order_by(db.text('date'))
 
+        # if the project's week is within next 2 weeks, send current week as well
+        now = utcnow().astimezone(self.timezone)
+
+        if self.schedule_start_at is not None:
+            current_week = Week.withdate(now)
+            schedule_start_week = Week.withdate(self.schedule_start_at)
+
+            if schedule_start_week > current_week and (schedule_start_week - current_week) <= 2:
+                if (schedule_start_week - current_week) == 2:
+                    session_dates.insert(0, (now + timedelta(days=7), 0))
```

Comment:
Something about this construct looks off. It needs a comment, and possibly cleaner nesting of `if` statements. However, this isn't a blocking issue.